src/main/java/seedu/trackermon/model/show/Status.java

```diff
@@ -3,19 +3,27 @@
 public enum Status {
 
     COMPLETED("completed"),
-    WATCHING("watching");
+    WATCHING("watching"),
+    PLANTOWATCH("plan-to-watch");
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Status should only be either completed or watching!";
+            "Status should only be completed or watching or plan to watch!";
+
+    private static final String PATTERN = "PLAN.*TO.*WATCH";
```

Comment:
What if the user type `s/plan/to/watch` or `s/plan /to/watch`? Just curious about whether it will break the system.

Reply:
That's the good point, i changed the pattern to not accept "/"~